### ISSUE TITLE
[Feature request]: Update interface address parser to support default WG configs

### ISSUE THREAD

**Issue by XFox111** [wiresock/WireSockUI#47]
It would be great if WireSockUI had Interface/Address support to be a plain IPv4 address (without network prefix)

## Justification
Default WireGuard configs have their Interface/Address set as plain IPv4 address without CIDR prefix:
```ini
[Interface]
Address = 10.15.15.7
PrivateKey = ABCDEF123456...
ListenPort = 51820
DNS = 10.15.15.1
...
```
But even though WireSock CLI is completely fine with it, WireSockUI requires it to include a CIDR prefix:
```ini
[Interface]
Address = 10.15.15.7/32
PrivateKey = ABCDEF123456...
ListenPort = 51820
DNS = 10.15.15.1
...
```
Besides, WireSockUI gives a pretty vague message when encountered the issue, which, if you don't know what the issue really is, can be quite frustrating:
```
"Address" in "Interface", invalid address "10.15.15.7"
```
Besides, it doesn't give me any info if I click on it:

<img width="455" alt="image" src="https://github.com/wiresock/WireSockUI/assets/28831743/712b775f-d16c-40dc-9f80-34c9776aa9df">

Plus, if I try to connect to the peer using system tray icon - it works just fine:

<img width="208" alt="image" src="https://github.com/wiresock/WireSockUI/assets/28831743/54dc47e9-fd39-4a42-a2e5-8f313f40b8bb">

## Alternatives
Alternatively, it would be great to make the error message more clear. I don't add new configs often, so when I do, I have a hard time remembering to add the prefix.

**Comment by wiresock** [comment]
Thank you for bringing this to my attention. WireSockUI incorporates a unique syntax checker that differs from the one used in WireSock VPN CLI. I will investigate this issue and aim to eliminate this limitation.

**Comment by wiresock** [comment]
I have addressed this issue in v0.2.2. Could you please give it a try?

**Comment by XFox111** [comment]
You are a rockstar! Thanks!
